guides/content/release_notes/3_7_0.md

```diff
@@ -182,6 +182,9 @@ of your own tips please submit a PR to help the next person!
 
   [Spark Solutions](https://github.com/spree/spree/pull/9146)
 
+* Deprecated `Spree::Order#remove_store_credit_payments` in favor of `Spree::Checkout::RemoveStoreCredit`
```

Comment:
@caudatecoder `Order#remove_store_credit_payments` without the `Spree` namespace is also fine :)